mule-maven-plugin/src/main/java/org/mule/tools/maven/utils/Exclusion.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.tools.maven.utils;
+
+public class Exclusion extends org.apache.maven.model.Exclusion implements org.mule.tools.api.packager.packaging.Exclusion {
```

Comment:
Are there tests for this class? 
I don't see it in the commit, could we please add if missing?